recipes/proj_datum/meta.yaml

```diff
@@ -0,0 +1,68 @@
+{% set europe_ver = "1.3" %}
+{% set north_america_ver = "1.2" %}
+{% set oceania_ver = "1.0" %}
+{% set world_ver = "1.0" %}
+
+package:
+  name: datumgrid
+
+source:
+  - url: http://download.osgeo.org/proj/proj-datumgrid-europe-{{ europe_ver }}.tar.gz
+    sha256: a6345a4a10c656ba436515c5890a67c7cc8a861a572cb562d03f3a228345a78f
+    folder: proj-datumgrid-europe
+
+  - url: http://download.osgeo.org/proj/proj-datumgrid-north-america-{{ north_america_ver }}.tar.gz
+    sha256: 00b51e15165e52f4ded7fd915ca41cdafa8a0d8e7fd12a65ba6a0ebe807f7ec0
+    folder: proj-datumgrid-north-america
+
+  - url: http://download.osgeo.org/proj/proj-datumgrid-oceania-{{ oceania_ver }}.tar.gz
+    sha256: 7cf0c3ca6221485317f1c752e592e9cb8b10eb9caf761e3847148fa3e4629a66
+    folder: proj-datumgrid-oceania
+
+  - url: http://download.osgeo.org/proj/proj-datumgrid-world-{{ world_ver }}.tar.gz
+    sha256: a488a5a69d1af6ec2ee83a2c64c52adac52e6dbfafe883f0341340009a9f40ba
+    folder: proj-datumgrid-world
+
+build:
+  number: 0
+
+requirements:
+  run:
+    - proj4
+
+test:
+  commands:
+    # TODO: we need real tests for each datumgrid.
+    - echo -105 40 | cs2cs +init=epsg:4326 +to +init=epsg:2975
```

Comment:
@hobu this is a dummy test. It would be nice if we could make one "Test CLI call" for each datumgrid. If you know something quick we can do here please let me know.